src/qililab/platform/platform.py

```diff
@@ -414,12 +414,13 @@ def get_element(self, alias: str):
             element = self.chip.get_node_from_alias(alias=alias)
         return element
 
-    def get_qrm_ch_id_from_qubit(self, alias: str, qubit_index: int) -> int:
+    def get_ch_id_from_qubit_and_bus(self, alias: str, qubit_index: int) -> int | None:
         """Finds a sequencer id for a given qubit given a bus alias. This is utility is added so that one can get a qrm's
```

Comment:
Just a small comment on the docstring
```suggestion
        """Finds a sequencer id for a given qubit given a bus alias. This utility is added so that one can get a qrm's
```